demo/vl-select.html

```diff
@@ -4,15 +4,69 @@
   <title>vl-select voorbeeld</title>
   <meta charset="UTF-8">
   <script type="module" src="demo.js"></script>
-  <script type="module" src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-  <script type="module" src="/node_modules/vl-ui-button/vl-button.js"></script>
+  <script type="module"
+          src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+  <script type="module"
+          src="/node_modules/vl-ui-button/vl-button.js"></script>
   <script type="module" src="../vl-select.src.js"></script>
+
+  <script type="text/javascript">
+    const demo = {
```

Comment:
Lijkt me zeker interessant om deze functies niet te dupliceren, maar we moeten hier wel een keuze maken of de onderstaande buttons (en hun gedrag) ook meenemen in de voorbeeld code of niet. Nu is het soms wel en soms niet...

Reply:
Ik heb ervoor gekozen om de demo code toch gelijk te trekken met het voorbeeld dat op de pagina te zien is. Mij lijkt dit iets duidelijker voor de ontwikkelaar. Laat maar weten als je het toch graag terug anders ziet.

Reply:
Alleen scripts die iets speciaal of extra doen zou ik opgenomen in de demo code, de API functies, wrappers of selectors die nodig zijn om een API functie te demonstreren lijken mij niet relevant.

Reply:
@Pieter-Beckers zoals je het nu opgelost hebt is prima, er zijn geen extra script tags nodig in de demo pagina. Zoals het voorheen was, vond ik niet goed omdat de demo pagina vervuild werd met een heleboel wrapper functies die irrelevant waren.